docs/src/layouts/BaseLayout.astro

```diff
@@ -44,8 +44,47 @@ const ogImage = `${Astro.site}og-banner-rocketsim.jpg`;
     <meta property="og:url" content={cannonicalUrl} />
     <meta name="twitter:card" content={ogImage} />
     <meta name="twitter:site" content="@rocketsim_app" />
+
+    <!-- Google Tag Manager -->
+    <script is:inline>
+      (function (w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != "dataLayer" ? "&l=" + l : "";
+        j.async = true;
+        j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, "script", "dataLayer", "GTM-59LFW4F4");
+    </script>
+    <!-- End Google Tag Manager -->
+
+    <!-- Plausible -->
+    <script
+      is:inline
+      defer
+      data-domain="rocketsim.app"
+      src="https://plausible.io/js/script.tagged-events.js"></script>
+    <script>
+      window.plausible =
+        window.plausible ||
+        function () {
+          (window.plausible.q = window.plausible.q || []).push(arguments);
+        };
+    </script>
+    <!-- End Plausible -->
```

Comment:
@AvdLee These scripts are added to all pages, including the privacy and terms pages. Is that okay?

Reply:
It is!